distribution/docs/src/main/resources/_contents/extending-contents.adoc

```diff
@@ -727,16 +727,16 @@ All bundles are then packaged into the KAR archive.
 	            same feature twice.
 	            Refer to Karaf forum posting at http://karaf.922171.n3.nabble.com/Duplicate-feature-repository-entry-using-archive-kar-to-build-deployable-applications-td3650850.html
 	            -->
-	            <resourcesDir>${project.build.directory}/doesNotExist</resourcesDir>
+	            <resourcesDir>${variable-prefix}project.build.directory}/doesNotExist</resourcesDir>
```

Comment:
Is that right, having {xxx}yyy} like that? Appears to be missing an open curly brace.